### ISSUE TITLE
Trying to create an event with a colon fails silently

### ISSUE THREAD

**Issue by MumblyFoo** [davish/obsidian-full-calendar#41]
**Describe the bug**
Cannot add new event in week view.  

Clicking "Save Event" does not close pop-up window or populate calendar with new event.  Create event pop-up remains visible after "Save Event" click.  Still able to edit event fields in pop-up so app not frozen.

Have tried clicking "Save Event' with blank title field and get an expected "Please fill out this field." error prompt.

**To Reproduce**
1. Open Calendar in Week view.
2. Click on "cell" for a day and time
3. Expected event definition pop-up appears.
4. Added Title, leaving everything else as default values.
5. Click on "Save Event"

**Expected behavior**
Expected "Save Event' click to close pop-up window and populate calendar.

**Screenshots**

**Console Errors**
Go to `View->Toggle Developer Tools` and open up the JavaScript console, reproduce your bug, and copy/paste any logs you see there.

Do not find the View->Toggle Developer Tools function in Obsidian.

**Device info
 - OS: Manjaro Linux 5.15.25-1 & KDE Plasma Version 5.24.2
 - Plugin Version 0.4.5
 - Obsidian Version v0.13.23

**Comment by brstrozy** [comment]
I have the same issue but only when my event title contains a ':' at least from what I can tell at the moment.

**Comment by MumblyFoo** [comment]
@brstrozy  I was beginning to think it was just my installation.  Might be useful to know your operating system and versions of Obsidian.

I'm hoping this can be fixed fairly soon.  I'm really looking forward to this plug-in.

**Comment by davish** [comment]
@brstrozy this is a limitation with Obsidian – note titles can't contain a `:` character, and since event titles are mirrored in note titles, this restriction is mirrored as well.

I'll work to add better error handling and notification here.

@MumblyFoo can you confirm that this only happens when you have a colon (`:`) in your event title? Otherwise this is a bigger bug and will require more investigation.

**Comment by MumblyFoo** [comment]
@davish before replying, I did a new install of Obsidian then installed FullCalendar from the Community Plug-in Repository.   Opened a calendar and created several new events without any problems.  My previous installation was done manually with the unzip process.  Any event title text caused the problem.   I don't believe I ever tried a title with a ":"  

Not sure if I messed up the installation earlier or if you've updated something.  I'm thrilled with this plugin and really appreciate all of the work you've done.  Congratulations on making the Community list!

Regards
